app/services/participants/profile_attributes.rb

```diff
@@ -25,7 +25,7 @@ def course_valid_for_participant
     def participant_has_user_profile
       return if errors.any?
 
-      errors.add(:participant_id, I18n.t(:invalid_participant)) if user_profile.blank?
```

Comment:
there are multiple reason we might not find a user_profile, one of them being the profile is withdrawn. 
Don't think this solves anything.
We need to re-write the whole validation process so that we can deliver meaningful error.